frontend/src/Form/beta/JsonSchemaForm.tsx

```diff
@@ -120,35 +107,38 @@ export default function JsonSchemaForm({
             ))}
           </List>
         </Stepper>
-      </div>
-      <Form
-        schema={currentStep.schema}
-        formData={currentStep.state}
-        onChange={onFormChange}
-        validator={validator}
-        widgets={widgets}
-        uiSchema={currentStep.uiSchema}
-        liveValidate={currentStep.shouldValidate}
-        omitExtraData
-        disabled={!canEdit}
-        liveOmit
-        formContext={{
-          editMode: canEdit,
-          formSchema: currentStep.schema,
-          defaultCurrentUser: defaultCurrentUserInEntityList,
-        }}
-        templates={
-          !canEdit
-            ? {
-                DescriptionFieldTemplate,
-                ArrayFieldTemplate,
-              }
-            : { ArrayFieldTemplate }
-        }
-      >
-        {/* eslint-disable-next-line react/jsx-no-useless-fragment */}
-        <></>
-      </Form>
-    </Stack>
+      </Grid>
+      <Divider orientation='vertical' flexItem sx={{ mr: '-1px' }} />
```

Comment:
An easier way of achieving this is to apply a `borderRight` to the left Grid item.

All children of a `<Grid container />` should be <Grid item>`s. That's why you've had to put a negative margin to make it work like this, because it's not designed to work that way.